@here/harp-mapview/lib/MapView.ts

```diff
@@ -490,6 +490,11 @@ export class MapView extends THREE.EventDispatcher {
      */
     defaultFontCatalog: string = DEFAULT_FONT_CATALOG;
 
+    /**
+     * Allows discarding the text rendering after the map rendering.
+     */
+    renderLabels: boolean = true;
```

Comment:
@zbigg @robertoraggi @amsgames94 I found it a convenient functionnality to only show the map in the effects example, it allows to not switch all the text techniques to `none` or flatout remove them. 

But it introduces a new API :sweat_smile: . Opinions?

Reply:
Yes and no. Bigger API ;) ?

Btw, You added new theme files, how do they differ from `base` ? Is it only effects ?

If yes, i would propose this solution:
```javascript
const theme = await ThemeLoader.loadAsync("...");
theme.effects = {
    ...
}
const newTheme = new ThemeVisitor(theme).visitStyles(style => {
    if (style.technique === "text") {
        style.technique = "none";
    }
})
const mapView = new MapView({theme: newTheme, ...)
```

Yes, this is code, but this shows clearly -  in example. How to add `effects` to map based on any style.
(Comment valid if we're customizing basic things).

Reply:
The new theme files (streets and outlines) should be quite different themes yes.